core-blocks/video/edit.js

```diff
@@ -23,10 +30,18 @@ class VideoEdit extends Component {
 		this.state = {
 			editing: ! this.props.attributes.src,
 		};
+
+		this.toggleAttribute = this.toggleAttribute.bind( this );
+	}
+
+	toggleAttribute( attribute ) {
```

Comment:
Seems like this would make a handy higher-order-component components could use as a sort of mixin, given it's the second place we've used this pattern (after `Audio`).

If you could either create a HOC for this and `Audio` (and probably other) components to use that'd be neat… or if that's too much of a pain could you at least file an issue with the "code quality" label for it and link to this comment? 😄

Reply:
Thanks for your comment. I am not a React expert (yet). So I created an issue for this: #7717